cmd/pathlist.go

```diff
@@ -0,0 +1,83 @@
+package cmd
+
+import 	(
+	"fmt"
+	"os"
+	"regexp"
+	"bufio"
+	"io"
+	"sort"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	regex = "data/[A-Za-z0-9_./]+\\.[a-zA-Z0-9]+"
+)
+
+var pattern regexp.Regexp
```

Comment:
There's no reason for this to be package-scoped; just move it to the pathlist scan cmd's `Run` function

Reply:
:boom: